### PR TITLE
[FE] Add Data Models to Bulk Uploader

### LINKED ISSUE #2285: Update Bulk Uploader with Data Models Summary
### Description

As an analyst, I want to use Bulk Uploader to upload Data Models and see a summary of models added or modified.

### Related Services

[See Backend services here](https://github.com/panther-labs/panther/pull/1945)

### Designs

Here is the[ RFC](https://github.com/panther-labs/panther-enterprise/tree/master/designs/security/simplify-detections)

Here are the [Designs](https://app.abstract.com/projects/558a8de6-7134-4c8b-91c5-62074bb1279b/branches/df9d59c8-eeb1-452d-a5f5-3712b64010be/collections/65b935ef-579f-4620-9530-c359b8d416e3)

### Acceptance Criteria

- When using Bulk Uploader, update the summary results to include a summary of Data Models that were uploaded after upload is complete
   - There should be a 4 summary box (next to Python Modules, Rules, Policies, and Data Models) with a summary of what's New, Modified, and Total

### PR DESCRIPTION
**Pull request by 3nvi**
## Background

This PR adds support for Data Models into the Bulk Uploader, while also fixing a small bug where the number of modified policies & rules was mix-matched.

Closes #2285 

## Screenshots

Screen Shot 2021-01-20 at 4 12 56 PM

## Changes

- Add Data Models results into the bulk uploader
- Fix "#modified" count inversion between rules & policies
- Rename `uploadPolicies` mutation to `uploadDetections`
- Update Tests

## Testing

- `npm run test`

### CODE REVIEW

**Comment by 3nvi** [comment]
@mattthetraveler @lindsey-w Would should test it tomorrow and add it to the release as a hotfix. The reason is that I spotted a bug in the bulk uploader where the number of modified rules actually matches the number of modified policies, so we need this PR eitherway just so it can fix this bug

File: web/src/components/wizards/BulkUploaderWizard/SuccessfulUploadPanel/SuccessfulUploadPanel.tsx [hunk 1]
```diff
@@ -53,9 +52,9 @@ const SuccessfulUpload: React.FC = () => {
               <Text fontWeight="bold">Rules</Text>
             </Flex>
             <Rows
-              newItems={uploadPolicies.newPolicies}
```
Comment (alexmylonas) [inline web/src/components/wizards/BulkUploaderWizard/SuccessfulUploadPanel/SuccessfulUploadPanel.tsx:56]: Nice catch!

**Review by vorillaz — APPROVED** [review]
Cool, thanks for this one Aggelos.

**Review by alexmylonas — APPROVED** [review]
LGTM! 🚢

**Pull request merged**